mne/dipole.py

```diff
@@ -1207,7 +1208,9 @@ def fit_dipole(evoked, cov, bem, trans=None, min_dist=5., n_jobs=1,
     # whitener[nzero, nzero] = 1.0 / np.sqrt(cov['eig'][nzero])
     # whitener = np.dot(whitener, cov['eigvec'])
 
-    whitener, rank = _get_whitener_data(info, cov, picks, verbose=False)
+    scalings = _handle_default('scalings', None)
```

Comment:
Same here, I doubt we really need to do this one. I'd consider it a bugfix not to, as it's a better match with what all other solvers do. Can you try not doing it and see if tests still pass? If they don't it might mean we have some problems with our cov rank scale factors.